src/ansys/geometry/core/tools/repair_tools.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (C) 2023 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Provides tools for repairing bodies."""
+
+from ansys.api.geometry.v0.repairtools_pb2 import (
+    FindDuplicateFacesRequest,
+    FindExtraEdgesRequest,
+    FindInexactEdgesRequest,
+    FindMissingFacesRequest,
+    FindSmallFacesRequest,
+    FindSplitEdgesRequest,
+    FindStitchFacesRequest,
+)
+from ansys.api.geometry.v0.repairtools_pb2_grpc import RepairToolsStub
+from beartype.typing import List
+from google.protobuf.wrappers_pb2 import DoubleValue
+
+from ansys.geometry.core.connection import GrpcClient
+from ansys.geometry.core.tools.problem_areas import (
+    DuplicateFaceProblemAreas,
+    ExtraEdgeProblemAreas,
+    InexactEdgeProblemAreas,
+    MissingFaceProblemAreas,
+    SmallFaceProblemAreas,
+    SplitEdgeProblemAreas,
+    StitchFaceProblemAreas,
+)
+
+
+class RepairTools:
+    """Repair tools for the pygeometry."""
+
+    def __init__(self, grpc_client: GrpcClient):
+        """Initialize Repair Tools class."""
+        self._grpc_client = grpc_client
+        self._repair_stub = RepairToolsStub(self._grpc_client.channel)
+
+    def find_split_edges(
+        self, ids: list[str], angle: float = 0.0, length: float = 0.0
```

Comment:
It is also preferred to use `Real` instead of `float`, see other classes for example